index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
```

Comment:
- [ ] kindly add the correct margins to the header, and the correct background color as shown in the Figma design


**your design:**

![image](https://user-images.githubusercontent.com/89282221/193342323-4eefc9ff-9161-40c3-bab5-e8c893b6a159.png)


**figma:**
![image](https://user-images.githubusercontent.com/89282221/193342251-8ffd74fc-37a8-425f-90d8-58898e20f7a7.png)